docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     depends_on:
       - db
   db:
-    image: genomenexus/gn-mongo:latest
+    build: https://github.com/inodb/genome-nexus-importer.git#add-exon-info-to-transcript
```

Comment:
points to `add-exon-info-to-transcript`, there's a fix in there to change `hgnc_symbol` to `hgnc_symbols` in the ensembl transcripts json used for import